sdk/core/azure-core/src/main/java/com/azure/core/http/policy/HttpLogOptions.java

```diff
@@ -279,4 +281,28 @@ public HttpLogOptions setResponseLogger(HttpResponseLogger responseLogger) {
         this.responseLogger = responseLogger;
         return this;
     }
+
+    /**
+     * Sets the flag that controls if header names which value is redacted should be logged.
+     * <p>
+     * Applies only if logging request and response headers is enabled. See {@link HttpLogOptions#setLogLevel(HttpLogDetailLevel)} for details.
+     * Defaults to `true` - redacted header names are logged.
```

Comment:
nit: since the default is to have this enabled, should we change the method name to `disableRedactedHeaderLogging`?